autoware_launch/config/planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/blind_spot.param.yaml

```diff
@@ -3,7 +3,8 @@
     blind_spot:
       use_pass_judge_line: true
       stop_line_margin: 1.0 # [m]
-      backward_length: 15.0 # [m]
+      backward_length: 50.0 # [m]
```

Comment:
@soblin 
Could you tell me the reason of this chage?

Reply:
If this value is 15m, and ego vehicle will detect collision with a bike only when (1) ego is within 15m from the stop line and (2) the bike is within 15m from the stop line and behind ego.
So ego vehicle need to completely stop while travelling less than 15m, at least 0.24G deceleration is required.
Maybe I should decide this value using pass judge length.